modules/API/pages/gsql-endpoints.adoc

```diff
@@ -4162,3 +4162,2356 @@ Response::
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X DELETE 'localhost:8123/gsql/v1/udt/tuples/TestTuple'
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----	
+{"error":false,"message":"Successfully dropped userDefinedTuple: [TestTup]."}
+----
+--
+====
+
+'''
+
+=== list all accumulators
+
+`GET /gsql/v1/udt/accum`
+
+This endpoint is used to get all accumulators.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X GET 'localhost:8123/gsql/v1/udt/accum'
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----		
+{"error":false,"message":"","results":[{"myGroup":"GroupByAccum<int a, SumAccum<int> b> myGroup\n"}]}
+----
+--
+====
+
+'''
+
+=== list one accumulator
+
+`GET /gsql/v1/udt/accum/{accumName}`
+
+This endpoint is used to get one accumulator by its name.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X GET 'localhost:8123/gsql/v1/udt/accum/myGroup'
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----		
+{"error":false,"message":"","results":[{"myGroup":"GroupByAccum<int a, SumAccum<int> b> myGroup\n"}]}
+----
+--
+====
+
+'''
+
+=== create accumulators
+
+`POST /gsql/v1/udt/accum`
+
+This endpoint is used to create one accumulator.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X POST 'localhost:8123/gsql/v1/udt/accum' -d '{"AccumString":"typedef GroupByAccum or typedef HeapAccum"}'
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----		
+{"error":false,"message":"Successfully create accumulator."}
+----
+--
+====
+
+'''
+
+=== delete mulitple accumulators
+
+`DELETE /gsql/v1/udt/accum?accumName=[all or accum1, accum2, ...]`
+
+This endpoint is used to delete multiple accumulators.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X DELETE 'localhost:8123/gsql/v1/udt/accum?accumName=myGroup'
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----		
+{"error":false,"message":"Successfully dropped accumulator: [myGroup]."}
+----
+--
+====
+===== Parameters:
+
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `accumName`
+| yes
+| the accumulator names to drop
+|===
+
+'''
+
+=== delete one accumulator
+
+`DELETE /gsql/v1/udt/accum/{accumName}`
+
+This endpoint is used to delete one accumulator.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X DELETE 'localhost:8123/gsql/v1/udt/accum/myGroup'
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----		
+{"error":false,"message":"Successfully dropped accumulator: [myGroup]."}
+----
+--
+====
+
+'''
+
+=== download expr files
+
+`GET /gsql/v1/udt/files/{fileName}`  fileName is one of: [tg_]ExprFunctions, [tg_]ExprUtil, TokenBank

```

Comment:
move `fileName is one of: [tg_]ExprFunctions, [tg_]ExprUtil, TokenBank` to below line:

```
This endpoint is used to download one file by its name, {fileName} can be one of: [tg_]ExprFunctions, [tg_]ExprUtil, TokenBank.
```